### DISCUSSION TITLE
Playback error 10.9.11

### DISCUSSION THREAD

**Discussion by madmex86** [jellyfin/jellyfin#12795] [General]
Hello,

my 4k Roku sticks updated to 2.1.6 and now I’m getting playback errors on some media. 

I tried to report it right now, but I kept getting an error submitting the report. 

Any ideas?

server is on a window 11 pc. 
server ver 10.9.11

I had no issues until today.

**Comment by crobibero** [comment]
Known issue, there is a new Roku app release scheduled for Thursday. 

https://github.com/jellyfin/jellyfin-roku/releases/tag/v2.1.7

**Reply by kevinsea** [reply]
Does Roku provide an ability to release to a beta group or have a Jellyfin 'release candidate' (RC) app?  I'd be happy to use the Roku RC app as my main Roku app to help test and use the production app as a fallback.

**Reply by crobibero** [reply]
Yes but not really, there is a limit of 20 users, and manually installing a build is simple enough that we stopped using beta channels

**Reply by kevinsea** [reply]
Yeah, 20 users doesn't work. What you really want is a canary release, right?  Are you able to release a "Jellyfin RC" separately in the store?  

(I run on lightweight hardware, so I disable video transcoding for all, that's why most of my content is offline, right?  My family is sitting in a couple of hotel rooms in FL looking to stream but can't. I think I can fix that for them, but I'd happily be an RC tester in the future to help find problems if there a way to do that with Roku)

**Comment by LRGEX** [comment]
> Hello,
> 
> my 4k Roku sticks updated to 2.1.6 and now I’m getting playback errors on some media.
> 
> I tried to report it right now, but I kept getting an error submitting the report.
> 
> Any ideas?
> 
> server is on a window 11 pc. server ver 10.9.11
> 
> I had no issues until today.

I have the same problem , playback error !! i don't know what to do, I think we should select the default player, before the update it was on client settings but now , the client settings is changed completely and there is no option for choosing default player like before.

**Comment by tkocou** [comment]
I've noticed that embedded subtitles can cause a player error. Purging the subtitle track or replacing it with a SRT file solves the playback problem. I would have registered an error report, but the "issues" area won't allow me to post the details in a proper report. Oh well.
